addons/web/static/src/views/calendar/calendar_renderer.scss

```diff
@@ -287,6 +293,17 @@
                 }
             }
 
+            .o_calendar_time_indicator_dot {
+                position: absolute;
+                height: .75rem;
+                width: .75rem;
+                border: 1px solid $o-view-background-color;
+                border-radius: $border-radius-pill;
+                transform: translate(-33%, -50%);
```

Comment:
I think it would look better with

```suggestion
                transform: translate(-50%, -50%);
```

However I do think Notion calendar provides a better solution to highlight the current day 
<img width="1145" alt="image" src="https://github.com/user-attachments/assets/c6a7c558-9672-4667-9c97-230762c78342" />

Since the width of the current day column is defined on the parent of your dot selector `td.fc-day`, I think you could easily achieve this result by reducing the opacity of the current hour line and apply a pseudo element to render the current day line on top, taking the width of its parent. 

Aside from this suggestion, it works great 😄